druid-shell/src/backend/wayland/window.rs

```diff
@@ -448,7 +459,8 @@ impl WindowData {
     /// - `size` is the physical size in pixels we are drawing.
     /// - `force` means draw the whole frame, even if it wasn't all invalidated.
     pub(crate) fn paint(&self, size: RawSize, buf: &mut [u8], force: bool) {
-        //log::trace!("Paint call");
+        self.handler.borrow_mut().prepare_paint();
```

Comment:
what issue is this attempting to fix? I didn't find this necessary in my fork, only painting problem I was running into was around animations.

Reply:
See the docs of `WinHandler::prepare_paint`. window handler is supposed to invalidate regions in prepare_paint. Without invalidation, there will be no repaints.

Reply:
I see. this doesn't seem to change any behavior rendering though. not against the change btw, was just curious because I didn't find it necessary and was curious what it fixes.

Reply:
oh I recall, this fixes invalidate example